packages/utils/src/assetData/baseAssets.ts

```diff
@@ -136,6 +136,21 @@ export const tcy: Readonly<Asset> = Object.freeze({
   relatedAssetKey: null,
 })
 
+export const maya: Readonly<Asset> = Object.freeze({
+  assetId: caip.mayaTokenAssetId,
+  chainId: caip.mayachainChainId,
+  name: 'MAYA',
+  networkName: 'MAYAChain',
+  symbol: 'MAYA',
+  precision: 4,
+  color: '#33FF99',
+  icon: 'https://raw.githubusercontent.com/shapeshift/web/feat_maya_asset/scripts/generateAssetData/thorchain/icons/maya.png',
```

Comment:
Apply this before merging me

```suggestion
  icon: 'https://raw.githubusercontent.com/shapeshift/web/develop/scripts/generateAssetData/thorchain/icons/maya.png',
```

Reply:
![image](https://github.com/user-attachments/assets/4e827a90-9b19-4aab-9a98-9358b10d3b1e)

found their content creators guidelines. we can use one of these colors if desired to be extra on point haha (same for cacao)

Reply:
Sounds good to me! 
- color: a24b364fad
- icon: 604fa239fd
- regen: 9c0da0450fbd7562b1855ebb7e3e433f5cd4ffef

<img width="1404" alt="image" src="https://github.com/user-attachments/assets/486dd50c-65c3-4a24-8e51-a6026cf045ad" />